chapters/en/chapter6/pre-trained_models.mdx

```diff
@@ -183,6 +183,94 @@ pre-trained checkpoint only supports English language:
 	height="450">
 </iframe>
 
+## Bark
+
+Bark is a transformer-based text-to-speech model proposed by Suno AI in [suno-ai/bark](https://github.com/suno-ai/bark).
+
+Bark is made of 4 main models:
+
+- `BarkSemanticModel` (also referred to as the 'text' model): a causal auto-regressive transformer model that takes as input tokenized text, and predicts semantic text tokens that capture the meaning of the text.
+- `BarkCoarseModel` (also referred to as the 'coarse acoustics' model): a causal autoregressive transformer, that takes as input the results of the `BarkSemanticModel` model. It aims at predicting the first two audio codebooks necessary for EnCodec.
+- `BarkFineModel` (the 'fine acoustics' model), this time a non-causal autoencoder transformer, which iteratively predicts the last codebooks based on the sum of the previous codebooks embeddings.
+- having predicted all the codebook channels from the `EncodecModel`, Bark uses it to decode the output audio array.
+
+It should be noted that each of the first three modules can support conditional speaker embeddings to condition the output sound according to specific predefined voice.
+
+Bark is an highly-controllable text-to-speech model, meaning you can use with various settings, as we are going to see.
+
+Before everything, load the model and its processor.
+
+The processor role here is two-sides:
+1. It is used to tokenize the input text, i.e. to cut it into small pieces that the model can understand.
+2. It stores speaker embeddings, i.e voice presets that can condition the generation.
+
+```python
+from transformers import BarkModel, BarkProcessor
+
+model = BarkModel.from_pretrained("suno/bark-small")
+processor = BarkProcessor.from_pretrained("suno/bark-small")
+```
+
+
+<Tip>
```

Comment:
I wouldn't write these all as tips, but rather give one example for each (as you did for the first Bark Colab you released) -> this makes it easier for the reader to understand how each tip relates to the code example